scipy/fft/_realtransforms.py

```diff
@@ -494,12 +509,16 @@ def dst(x, type=2, n=None, axis=-1, norm=None, overwrite_x=False, workers=None):
 
     Notes
     -----
-    For a single dimension array ``x``.
+    .. warning:: For ``type in {2, 3}``, ``norm="ortho"`` breaks the direct
+                 correspondence with the inverse direct Fourier transform.
+
+    For ``norm="ortho"```` both the `dst` and `idst` are made orthogonal through
```

Comment:
```suggestion
    For ``norm="ortho"`` both the `dst` and `idst` are made orthogonal through
```

Reply:
good catch. there was one more instance higher up, I pushed a fix for both